.github/workflows/nuget-publish.yml

```diff
@@ -2,7 +2,7 @@ name: Release Package Version
 on:
   push:
     tags:
-     - 2.*     
+     - 3.*     
```

Comment:
```suggestion
     - *.*     
```

I wonder if we don't just make it valid for any tag?

Reply:
Yeah, that should work.